### ISSUE TITLE
[Feature Request] CTRL+A (Select All) Expand selection to multiple columns

### ISSUE THREAD

**Issue by bryc** [squiggythings/WaveTracker#12]
Given the following screenshot:

![image](https://github.com/user-attachments/assets/29395c1b-215a-47a0-8feb-ce643997cdae)

Pressing CTRL+A will select the entire middle column (because of the cursor). 

Pressing CTRL+A twice will select all columns.

But given that the initial selection spans 3 columns, I think it would be very useful to allow CTRL+A to perform Select All for all 3 selected columns.  Could be a big time save!

**Comment by darkhog** [comment]
Yeah, that would be nice. Then if you ACTUALLY want to select all columns, you could press CTRL+A again to also select the rest.
